src/pages/category/Category.jsx

```diff
@@ -0,0 +1,71 @@
+import { useEffect } from "react";
+import { Watch as Loader } from "react-loader-spinner";
+import { useDispatch, useSelector } from "react-redux";
+import { useNavigate, useParams } from "react-router-dom";
+import { getCategories, getQuizesInCategory } from "reducers/categorySlice";
+import "./category.css";
+
+export const Category = () => {
+  const { categoryId } = useParams();
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const { loading, categories, quizes } = useSelector(
+    (state) => state.categories
+  );
+  const category = categories.filter((c) => c._id === categoryId)[0];
+
+  useEffect(() => {
+    if (categoryId) {
+      dispatch(getQuizesInCategory(categoryId));
+      if (categories.length === 0) {
+        dispatch(getCategories());
+      }
+    }
+  }, []);
+
+  return (
+    <div className="category_page_wrapper">
+      {categories.length !== 0 && quizes.length !== 0 && (
+        <div className="landing_wrapper flex_column flex_gap2">
+          <div className="title_wrapper flex_column flex_align_center flex_gap2">
+            <div className="back_title_wrapper flex_row flex_align_center flex_justify_center">
+              <div className="category_cta flex_row flex_align_center flex_gap2">
+                <button
+                  className="sui_btn btn_v1 flex_row flex_align_center flex_gap1"
+                  onClick={() => navigate("/")}
```

Comment:
Suggestion: navigate(-1) goes back ??

Reply:
yes it does, but it will not work for the case when you open the category page directly from a link.